src/main/java/wolox/training/models/Book.java

```diff
@@ -1,8 +1,14 @@
 package wolox.training.models;
 
-import javax.persistence.Column;
+import javax.persistence.*;
```

Comment:
We do not like wildcards. You can configure your IntelliJ: https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-